### DISCUSSION TITLE
Change ISO File for personal needs

### DISCUSSION THREAD

**Discussion by db5fp** [finnix/finnix#43] [Q&A]
Hello,

I need to reset 200 PC, more precisely delete the hard disk, I want to do this with hdparm and ATA-SECURE-ERASE. For this I would like to add a script to squashfs and preferably have it executed immediately during the boot process. 
However, after changing the squashfs, I have not been able to add them back into the ISO file so that the PC will boot again. Is there a simple procedure I can use to change the image?

Thanks a lot

**Comment by likuilin** [comment]
You could use the `systemd.run` kernel command line argument to run hdparm, and not have to modify the squashfs. With that and `toram` you can remove the flash drive after boot, and safely move on to the next PC after everything's copied even, before the ATA erase finishes.
